crates/valence_entity/build.rs

```diff
@@ -421,23 +464,98 @@ fn build() -> anyhow::Result<TokenStream> {
                             _ => {}
                         }
                     }
-                    MarkerOrField::Field { entity_name, field } => {
+                    MarkerOrField::Field {
+                        entity_name: entity_name_2,
+                        field,
+                    } => {
                         let snake_field_name = field.name.to_snake_case();
                         let pascal_field_name = field.name.to_pascal_case();
                         let pascal_field_name_ident = ident(&pascal_field_name);
-                        let stripped_entity_name = strip_entity_suffix(entity_name);
-                        let stripped_snake_entity_name = stripped_entity_name.to_snake_case();
-                        let stripped_snake_entity_name_ident = ident(&stripped_snake_entity_name);
+                        let stripped_entity_name = strip_entity_suffix(entity_name_2);
+                        let stripped_snake_entity_name_1 = stripped_entity_name.to_snake_case();
+                        let stripped_snake_entity_name_1_ident =
```

Comment:
Do we need these variables with the `_1` and `_2` suffixes? I would prefer something more descriptive.